src/platform/run_return.rs

```diff
@@ -33,7 +34,12 @@ pub trait EventLoopExtRunReturn {
     /// underlying OS APIs, which cannot be hidden by `winit` without severe stability repercussions.
     ///
     /// You are strongly encouraged to use `run`, unless the use of this is absolutely necessary.
-    fn run_return<F>(&mut self, event_handler: F)
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Unix-alikes** (**X11** or **Wayland**): This function returns `1` upon disconnection from
+    ///   the display server.
```

Comment:
It might be, but I'm not too sure how to word that.

```rust
    /// - **Unix-alikes** (**X11** or **Wayland**): The function exits with exit code 1 if the display server disconnects.
```
sounds a bit weird imo, since the function seemingly never "exits".

Reply:
Pretty fine, yeah. Thanks!